### ISSUE TITLE
Create a climate change data explainer?

### ISSUE THREAD

**Issue by see24** [LandSciTech/ccviR#184]
The tool is somewhat climate data agnostic but it would be good to define more clearly how CMD is calculated and where it comes from. There are different ways to calculate it but does our tool require one?

**Comment by steffilazerte** [comment]
Would this be related to #206 ?

**Comment by see24** [comment]
Yes this could potentially be a more general climate data vignette that explains where all the climate data comes from. It should be targeted at a user of the CCVI app that explains where the data comes from without the user needing all the details of how the climate data prep app works.

**Comment by see24** [comment]
This is a suggestion coming out of the other contract that we had going on so I wasn't expecting it to be one of your deliverables. But if you can keep this end product in mind as you are writing up documentation on the CCEI that would be great
